i18n/nl/fotema.ftl

```diff
@@ -0,0 +1,230 @@
+# Photo/Video Information Sidebar
+# Audio codec, such as "OPUS".
+infobar-audio-codec = Audiocodec
+# Photo/Video Information Sidebar
+# Width and height of photo or video.
+infobar-dimensions = Afmetingen
+# Preferences
+# Selfies page enabled or disabled.
+# Attributes:
+#   .subtitle - Description of toggle button action action.
+prefs-views-selfies = Selfies
+    .subtitle = Toon een aparte sectie voor selfies die op iOS-apparaten gemaakt zijn. Herstart { -app-name } om de wijziging toe te passen.
+# Preferences
+# Title of section of preferences for views
+prefs-views-section = Zijpaneelsecties
+    .description = Toon of verberg zijpaneelsecties
+# Progress bar for background tasks
+# Extracting details from photo EXIF data
+progress-metadata-photos = Bezig met verwerken van metagegevens…
+# Progress bar for background tasks
+# Extracting details from video container metadata
+progress-metadata-videos = Bezig met verwerken van metagegevens…
+# Progress bar for background tasks
+# Generating thumbnails from photos
+progress-thumbnails-photos = Bezig met maken van fotominiaturen…
+# Progress bar for background tasks
+# Generating thumbnails from videos
+progress-thumbnails-videos = Bezig met maken van videominiaturen…
+# Progress bar for background tasks
+# Transcoding videos to a compatible format
+progress-convert-videos = Bezig met omzetten…
+# Progress bar for background tasks
+# Extracting motion photo videos
+progress-motion-photo = Bezig met verwerken van actiefoto's…
+# Progress bar for background tasks
+# Not doing any background work
+progress-idle = Inactief.
+# Notification banner for background tasks
+# Scanning file system for new photos
+banner-scan-photos = Bezig met zoeken naar foto's…
+# Notification banner for background tasks
+# Scanning file system for new videos
+banner-scan-videos = Bezig met zoeken naar video's…
+# Thumbnail decorations
+# Label on month album thumbnails.
+# Variables:
+#   $month - month number (1 through 12).
+#   $year - year e.g., 2024
+# Translator note: do not values in square brackets, such as '[other]'.
+month-thumbnail-label =
+    { $month ->
+        [1] januari { $year }
+        [2] februari { $year }
+        [3] maart { $year }
+        [4] april { $year }
+        [5] mei { $year }
+        [6] Juni { $year }
+        [7] juli { $year }
+        [8] augustus { $year }
+        [9] september { $year }
+        [10] oktober { $year }
+        [11] november { $year }
+        [12] december { $year }
+       *[other] { $year }
+    }
+# About Dialog
+# Section header for open source projects acknowledgements.
+about-opensource = Opensourceprojecten
+# Photo/Video Viewer
+# Go to previous button when viewing photo or video.
+# Attributes:
+#  .tooltip - Tooltip on mouse hover.
+viewer-previous =
+    .tooltip = Vorige
+# About Dialog
+# Translator note: add one translator per-line to get a translation
+# credit in the Fotema's "About" page.
+about-translator-credits = David Bliss <>
```

Comment:
I didn't translate this, you did! Feel free to put your name and details here and it will appear somewhere in the "About" dialog.